lib/strings/text.php

```diff
@@ -14,36 +14,36 @@ public function __construct(string $str)
 
     public function isEmpty():bool
     {
-        if (!empty($this->str))
+        if (empty($this->str))
```

Comment:
how about a one liner 

`return empty(this->str)?true:false;`

Reply:
resolved